tests/test_crossdock.py

```diff
@@ -48,7 +51,7 @@ def tracer():
         service_name='test-tracer',
         sampler=ConstSampler(True),
         reporter=InMemoryReporter(),
-        scope_manager=TornadoScopeManager()
+        scope_manager=scope_manager() if scope_manager else None
```

Comment:
```suggestion
        scope_manager=scope_manager and scope_manager()
```

Reply:
why not just instantiate it next to the import?